src/app/components/feed/jh-main-feed-info.html

```diff
@@ -1 +1,15 @@
-<span class="feed-info">{{ vm.name() }}</span>
+<span>
+  <div class="name-edit"
+  ng-show="vm.isEditing()">
+    <input ng-keypress="vm.keyDetect($event)"
+      ng-model="vm.newName">
+    <span class="glyphicon glyphicon-question-sign"
+      title="Press Enter to change &#013; Press Escape to keep"></span>
+  </div>
+  <span class="feed-info"
+  title="Double Click to Edit Name"
+  ng-dblclick="vm.makeNameEditable()"
+  ng-hide="vm.isEditing()">
+  {{ vm.name() }}
+  </span>
+</span>
```

Comment:
A div `class="name-edit"` has been added here. It shows only when user double clicks on the name. There is an input field inside it that allows users to change name. The span inside this div is a `help` icon. When a user hover over this icon, it shows the instructions for changing name.

The `feed-info` span shows only when user is not editing.